src/screens/searchResult/screen/tabs/topics/container/topicsResultsContainer.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import get from 'lodash/get';
+import { withNavigation } from 'react-navigation';
+import { connect } from 'react-redux';
+
+import ROUTES from '../../../../../../constants/routeNames';
+
+import { getTrendingTags } from '../../../../../../providers/hive/dhive';
+import { getLeaderboard } from '../../../../../../providers/ecency/ecency';
+import { isCommunity } from '../../../../../../utils/communityValidation';
+
+const OtherResultContainer = (props) => {
+  const [tags, setTags] = useState([]);
+  const [noResult, setNoResult] = useState(false);
+
+  const { children, navigation, searchValue, username } = props;
+
+  useEffect(() => {
+    setNoResult(false);
+    setTags([]);
+
+    if (searchValue.length > 10) {
+      setNoResult(true);
```

Comment:
if string is longer than 10, we can still show results if we have but don't do search for longer text.
for example, if I search `longertexts` which is 11 character, show if there are results for `longertext` first 10 characters of that string.